src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -0,0 +1,73 @@
+package cromwell.engine
+
+import akka.actor.{Actor, ActorRef}
+import akka.actor.SupervisorStrategy.Stop
+import akka.event.{LoggingReceive, Logging}
+import cromwell.binding.{Call, WdlValue}
+import cromwell.engine.CallActor._
+import cromwell.engine.WorkflowActor.InvalidOperation
+
+import scala.language.postfixOps
+import scala.util.{Failure, Success}
+
+
+object CallActor {
+
+  case class Start(call: Call, symbolTable: SymbolTable)
+
+  case class Run(commandLine: String)
+
+  case object Started
+
+  case class Done(outputs: Map[String, WdlValue])
+
+  case class Failed(throwable: Throwable)
+
+  case object Stopped
+}
+
+/**
+ * Probably not the right level of abstraction for a system that supports richer
+ * control structures like loops, conditionals, scatter etc., but this should be
+ * sufficient for Sprint 2/3.
+ */
+class CallActor extends Actor {
+
+  val log = Logging(context.system, this)
+
+  var _creator: ActorRef = _
+  var _started: Boolean = false
+
+  override def receive: Receive = {
+    LoggingReceive {
+      case Start(call, symbolTable) =>
```

Comment:
The reason why I say that is because I'd be nice if third parties could build on top of the `cromwell.binding` layer to implement their own engines and it'd be nice if we kept that API clear of any methods/data structures that happen to be used in our engine.